.github/actions/pnpm/action.yml

```diff
@@ -0,0 +1,14 @@
+name: Setup node and pnpm
```

Comment:
Should we make a repo and publish this action so it doesn't need to be created in every addon?

Reply:
that'd be helpful, I can make such an action